syntax = "proto3";

package security;

option java_multiple_files = true;
option java_package = "common.model.protos";
option java_outer_classname = "SecurityTypeProtos";

enum SecurityTypeProto {
  UNKNOWN_SECURITY_TYPE = 0; //Maps to Security
  CASH_SECURITY = 1;
  EQUITY_SECURITY = 2;
  BOND_SECURITY = 3;
  TIPS = 4;
  FRN = 5;
}
